governance/communications/maker-realy-ep-07.md

```diff
@@ -0,0 +1,267 @@
+# Maker Relay 07
```

Comment:
Filename is spelt incorrectly. maker-realy-ep-07.md  -> maker-relay-ep-07.md

Reply:
looks like 7 is missing. Episode zero is the one we didn't deploy on time to the forums or reddit. A dry-run relay. 

![Screen Shot 2020-08-24 at 12 12 50 PM](https://user-images.githubusercontent.com/8991325/91086033-2b104b80-e603-11ea-8c48-6fb5981ae95f.png)

Reply:
i deleted it on the branch. Hopefully that fixes it. 🙏